terraform/aks/variables.tf

```diff
@@ -0,0 +1,58 @@
+variable "cluster" {
+  description = "AKS cluster where this app is deployed. Either 'test' or 'production'"
+}
+variable "namespace" {
+  description = "AKS namespace where this app is deployed"
+}
+variable "environment" {
+  description = "Name of the deployed environment in AKS"
+}
+variable "azure_credentials_json" {
+  default     = null
+  description = "JSON containing the service principal authentication key when running in automation"
+}
+variable "azure_resource_prefix" {
+  description = "Standard resource prefix. Usually s189t01 (test) or s189p01 (production)"
+}
+variable "config_short" {
+  description = "Short name of the environment configuration, e.g. dv, st, pd..."
+}
+variable "service_name" {
+  description = "Full name of the service. Lowercase and hyphen separated"
+}
+variable "service_short" {
+  description = "Short name to identify the service. Up to 6 charcters."
+}
+variable "deploy_azure_backing_services" {
+  default     = true
+  description = "Deploy real Azure backing services like databases, as opposed to containers inside of AKS"
+}
+variable "enable_postgres_ssl" {
+  default     = true
+  description = "Enforce SSL connection from the client side"
+}
+variable "docker_image" {
+  description = "Docker image full name to identify it in the registry. Includes docker registry, repository and tag e.g.: ghcr.io/dfe-digital/teacher-pay-calculator:673f6309fd0c907014f44d6732496ecd92a2bcd0"
```

Comment:
Could this be updated for the local project?

Reply:
Docker image is not required right now